CSLabs.ConsoleUtil/Commands/ListHypervisors.cs

```diff
@@ -14,8 +14,10 @@ public class ListHypervisorsOptions
     
     public class ListHypervisorsCommand : AsyncCommand<ListHypervisorsOptions>
     {
-        private DefaultContext _context;
-        private string _encryptionKey;
+        private readonly DefaultContext _context;
+        
+        // Do we need this field? It's totally unread, only ever set in the constructor but never used
+        private readonly string _encryptionKey;
```

Comment:
@jasekiw @Cliftonz can we delete this property? It's never read after instantiation, the only reference to it is in the constructor.

Reply:
That would be a jason question.

Reply:
I do know that we will be changing from passwords to apikeys soon when communicating to proxmox.

Reply:
Yeah that was why I wanted to check with you guys, I wasn't sure if there were future plans to use this property or not, so I wanted to check in before I just removed it. Sonarcloud isn't happy with it being here, but I can mark that code smell as resolved if we have plans to use it soon.

Reply:
It think this was accidentally copied from another command, it can be deleted.